docs/pressable.md

```diff
@@ -96,81 +99,83 @@ const styles = StyleSheet.create({
     backgroundColor: '#f9f9f9'
   }
 });
+
+export default App;
 ```
 
 ## Props
 
-### `android_disableSound`
+### `android_disableSound` <div class="label android">Android</div>
 
 If true, doesn't play Android system sound on press.
 
-| Type    | Required | Platform |
-| ------- | -------- | -------- |
-| boolean | No       | Android  |
+| Type    | Required | Default |
+| ------- | -------- | ------- |
+| boolean | No       | `false` |
 
-### `android_rippleColor`
+### `android_rippleColor` <div class="label android">Android</div>
 
 Enables the Android ripple effect and configures its color.
 
-| Type                                         | Required | Platform |
-| -------------------------------------------- | -------- | -------- |
-| [color](https://reactnative.dev/docs/colors) | No       | Android  |
+| Type                                         | Required |
+| -------------------------------------------- | -------- |
+| [color](https://reactnative.dev/docs/colors) | No       |
 
 ### `children`
 
 Either children or a function that receives a boolean reflecting whether the component is currently pressed.
 
-| Type       | Required |
-| ---------- | -------- |
-| React.Node | No       |
+| Type                     | Required |
+| ------------------------ | -------- |
+| [React Node](react-node) | No       |
 
 ### `delayLongPress`
 
-Duration (in milliseconds) from `onPress` before `onLongPress` is called.
+Duration (in milliseconds) from `onPressIn` before `onLongPress` is called.
 
 | Type   | Required | Default |
 | ------ | -------- | ------- |
-| number | No       | 500     |
+| number | No       | 370     |
 
 ### `disabled`
 
 Whether the press behavior is disabled.
 
-| Type    | Required |
-| ------- | -------- |
-| boolean | No       |
+| Type    | Required | Default |
+| ------- | -------- | ------- |
+| boolean | No       | `false` |
 
 ### `hitSlop`
 
 Sets additional distance outside of element in which a press can be detected.
 
-| Type       | Required |
-| ---------- | -------- |
-| RectOrSize | No       |
+| Type                                 | Required |
+| ------------------------------------ | -------- |
+| [`RectOrSize`](rectorsize) or number | No       |

```

Comment:
`or number` is redundant. That is what RectOrSize means :)

Reply:
How would you word the `RectOrSize` page though? It doesn't quite work right as a reference page like that...

Reply:
Maybe make it a Rect page and say `Rect or number` here?